Handler errors with Gaffe in production only

j15e opened the pull request:
Sinon on voit pas les erreurs avec better_errors en dev eh!

j15e committed:
Handler errors with Gaffe in production only

garno commented:
Tu es sûr du comportement? Chez moi, better_errors semble prendre le dessus sur Gaffe.

https://github.com/mirego/gaffe#rails-development-mode

j15e commented:
Pas dans tous les cas je crois, si tu as un erreur dans le routes.rb
mettons Gaffe me retourne une erreur moi en dev (avec 0 détails)

j15e committed:
Add config/puma.rb & set single thread in dev for better_errors

https://github.com/charliesome/better_errors#unicorn-puma-and-other-multi-worker-servers

j15e commented:
Je pense qu'il y a anguille sous roche, il autre chose je crois qui marche pas avec better_errors et rails 4 et/ou le setup que j'ai fait. J'ai pas trouvé nul part une mention de rails 4 sur le projet de better_errors, c'est peut-être là le vrai problème.

remi commented:
@j15e Peux-tu enlever ton code relié à puma de cette pull request et en faire une autre? À moins que ça soit relié avec Gaffe vs. better_errors mais je ne verrais pas pourquoi.
